index.scss

```diff
@@ -106,13 +106,13 @@ p {
 .language-json { 
   .token.string,
   .token.number {
-    color: var(--json-purple) !important;
+    color: #e6db74 !important;
```

Comment:
add color from variables, why remove var color?

Reply:
Because it is unique color that is used only once so creating a variable would be unnecessary